packages/core/test/oracle/Finder.js

```diff
@@ -11,8 +12,8 @@ contract("Finder", function(accounts) {
   it("General methods", async function() {
     const finder = await Finder.deployed();
 
-    const interfaceName1 = web3.utils.hexToBytes(web3.utils.utf8ToHex("interface1"));
-    const interfaceName2 = web3.utils.hexToBytes(web3.utils.utf8ToHex("interface2"));
```

Comment:
I guess truffle or web3 no longer casts `bytes` correctly anymore to pass into a method like `changeImplementationAddress` which expects a `bytes32` argument?